backend/src/middleware/notifications/notificationsMiddleware.spec.js

```diff
@@ -541,5 +571,148 @@ describe('notifications', () => {
         })
       })
     })
+
+    describe('given I file a report on a', () => {
+      let resourceId
+      const reasonCategory = 'discrimination_etc'
+      const reasonDescription = 'I am free to be gay !!!'
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T18:35:36Z" title="Thursday, January 23rd 2020, 7:35:36 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -541,5 +573,148 @@ describe('notifications', () => {
         })
       })
     })
+
+    describe('given me filing a report of a', () => {
+      let resourceId
+      const reasonCategory = 'discrimination_etc'
+      const reasonDescription = 'I am free to be gay !!!'
```


you certainly are, did someone say you're not?!

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-24T09:22:08Z" title="Friday, January 24th 2020, 10:22:08 am +01:00">Jan 24, 2020</time>_

---


Yes I am!